src/assets/i18n/en_US.json

```diff
@@ -405,6 +408,55 @@
   "account.quotes.widget.responded.label": "Responded quotes",
   "account.quotes.widget.submitted.label": "Sent requests",
   "account.quotes.widget.view_all_quotes.link": "View all quotes",
+  "account.recurring_order.delete.confirmation": "Your recurring order has been deleted.",
+  "account.recurring_order.delete_dialog.header": "Delete recurring order \"{{0}}\"",
+  "account.recurring_order.details.active.text": "Active",
+  "account.recurring_order.details.additional_info.heading": "Additional information",
+  "account.recurring_order.details.approved.text": "approved by {{name}} on {{date}}",
+  "account.recurring_order.details.billing_address.heading": "Invoice address",
+  "account.recurring_order.details.breadcrumb": "Recurring order",
+  "account.recurring_order.details.buyer_info.heading": "Buyer",
+  "account.recurring_order.details.cost_center.label": "Cost center",
+  "account.recurring_order.details.creation_date.label": "Creation date",
+  "account.recurring_order.details.expired.text": "Ended",
+  "account.recurring_order.details.inactive-by-system.message": "The last planned order could not be placed and the recurring order has been deactivated.",
+  "account.recurring_order.details.inactive.message": "Your recurring order is currently inactive. Activate it now to continue receiving regular deliveries and exclusive discounts.",
+  "account.recurring_order.details.inactive.text": "Inactive",
+  "account.recurring_order.details.info_message": "Please note: Orders are created even if prices or the availability of subscribed items have changed. See the order confirmation for details.",
```

Comment:
As we want to get rid of the word "subscription":  I'd suggest changing "of subscribed items" to "of included items" here.